assets/queries/cloudFormation/ec2_subnet_unable_connect_cluster/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "ec2_vulnerable_public_subnet",
+  "queryName": "EC2 vulnerable public subnet",
```

Comment:
Change query name to:
`EC2 subnet mapping public IP on launch`

Change directory name to:
`ec2_subnet_mapping_public_ip_on_launch`

Reply:
Also change the id accordingly